cost/azure/savings_plan/utilization/azure_savings_plan_utilization.pt

```diff
@@ -0,0 +1,501 @@
+      }
+
+      sp = _.omit(sp, "utilization")
+    }
+ 
+    return sp
+  })
+  EOS
+end
+
+# ORGANIZE DATA FOR POLICY INCIDENT
+datasource "ds_sp_data_for_incident" do
+  run_script $js_sp_data_for_incident, $ds_filtered_savings_plans, $ds_currency_reference, $param_utilization_threshold
+end
+
+script "js_sp_data_for_incident", type: "javascript" do
+  parameters "ds_filtered_savings_plans", "ds_currency_reference", "param_utilization_threshold"
+  result "result"
+  code <<-EOS
+
+  // function for formatting spend number
+  function formatNumber(number, separator) {
+    var numString = number.toString();
+    var values = numString.split(".");
+    var result = ''
+    while ( values[0].length > 3 ) {
+      var chunk = values[0].substr(-3)
+      values[0] = values[0].substr(0, values[0].length - 3)
+      result = separator + chunk + result
+    }
+    if ( values[0].length > 0 ) {
+      result = values[0] + result
+    }
+    if ( values[1] == undefined ) {
+      return result;
+    }
+    return result + "." + values[1];
+  }
+
+  // format costs with currency symbol and thousands separator
+  var cur = "", separator = ""
+  if ( _.size(ds_filtered_savings_plans) > 0 && ds_filtered_savings_plans[0].commitmentCurrency !== undefined ) {
+    if ( ds_currency_reference[ ds_filtered_savings_plans[0].commitmentCurrency ] !== undefined ) {
+      cur = ds_currency_reference[ ds_filtered_savings_plans[0].commitmentCurrency ].symbol
+      if ( ds_currency_reference[ ds_filtered_savings_plans[0].commitmentCurrency ].t_separator !== undefined ) {
+        separator = ds_currency_reference[ ds_filtered_savings_plans[0].commitmentCurrency ].t_separator
+      } else {
+        separator = ""
+      }
+    } else {
+      cur = ""
+      separator = ""
+    }
+  } else {
+    cur = "$"
+    separator = ","
+  }
+
+  filtered_sps = _.reject(ds_filtered_savings_plans, function(sp) {
+    if (sp.utilizationAmount != null) {
+      return Number(sp.utilizationAmount) > param_utilization_threshold
+    } else {
+      return sp
+    }
+  })
+
+  total_wastage = 0
+  _.each(filtered_sps, function(sp) {
+    if (sp.commitmentAmount && sp.commitmentAmount != null) {
+      total_wastage += Number(sp.commitmentAmount)
+    }
+  })
+
+  total_wastage = cur + " " + formatNumber((Math.round(total_wastage * 100) / 100), separator)
+  
+  result = {
+    "savings_plan_details": filtered_sps,
+    "message": "The total estimated hourly wasted spend is " + total_wastage
+  }
+  EOS
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "pol_azure_sp_utilization" do
+  validate $ds_sp_data_for_incident do
+    summary_template "Azure Savings Plan Utilization Details: {{ len data.savings_plan_details }} Azure Savings Plans - Last {{ parameters.param_lookback_days}} days"
```

Comment:
Summary template should include a reference to the applied policy name, similar to other policy templates. This is to assist users in finding the correct incident if they apply the policy template multiple times with different names.

Reply:
![image](https://github.com/user-attachments/assets/6d6ed9b5-1b83-45b4-81ce-c21e80fcd235)

Reference to Applied Policy Name in Policy Incident title has been added